lib/PlaidClient.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+var Promise = require('bluebird');
```

Comment:
rather than shadowing `Promise`, it would be more idiomatic to give the bluebird promise implementation a different name.

canonically @ plaid, we use `P`

```node
var P = require('bluebird')
```